### PR TITLE
Compatibility data: coinbase conio copay edge

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

**Comment by jnewbery** [comment]
rebased

**Comment by jnewbery** [comment]
@adamjonas - please review for typos/formatting/etc. Let me know if you can't review this week. Thanks!

File: _data/compatibility/coinbase.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        Bumping RBF Transaction - After a period of time the View Transaction link shows up.
+    - image: /img/compatibility/coinbase/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction list. No RBF label.
+    - image: /img/compatibility/coinbase/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction details. No RBF label. Further transaction details are at blockcypher explorer which does not label RBF transactions.
+    - image: /img/compatibility/coinbase/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - After bumped transaction confirmed, the bumped transaction then shows up and is credited. Original transactions stays as pending. (even after 100 confirmations)
```
Comment (adamjonas) [inline _data/compatibility/coinbase.yaml]: replace: Original transactions stays as pending. (even after 100 confirmations)
with: Original transactions stay as pending (even after 100 confirmations).

File: _data/compatibility/edge.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        Sending RBF Transaction - Mining fee custom dialog.
+    - image: /img/compatibility/edge/transaction-details-sent.png
+      caption: >
+        Bumping RBF Enabled Transaction - Mining fee custom dialog.
+    - image: /img/compatibility/edge/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Receiving RBF enabled transaction. No RBF Flag.
+    - image: /img/compatibility/edge/transaction-details-incoming-rbf-advanced.png
+      caption: >
+        Receiving RBF Transaction - Transaction details for received RBF transaction. Links to blockchair.com explorer.
```
Comment (adamjonas) [inline _data/compatibility/edge.yaml]: blockchair -> blockchain

File: _data/compatibility/edge.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    - image: /img/compatibility/edge/segwit/receive-screen.png
+      caption: >
+        Edge generates only 
+    - image: /img/compatibility/edge/segwit/send-screen.png
+      caption: >
+        Edge allows sends to both segwit v0 and v1 native bech32 addresses. 
+    - image: /img/compatibility/edge/segwit/transaction-list-segwit-v1-pending.png
+      caption: >
+        Balance is calculated incorrectly after segwit v1 transaction is "sent".
+        Subsequent transactions out of the wallet do not seem to be broadcast.
```
Comment (adamjonas) [inline _data/compatibility/edge.yaml]: be broadcasted

File: _data/compatibility/copay.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        Sending RBF Transaction - Dialog box for custom fee level
+    - image: /img/compatibility/copay/transaction-list-sent.png
+      caption: >
+        Bumping RBF Enabled Transaction - Transactions list
+    - image: /img/compatibility/copay/transaction-details-sent.png
+      caption: >
+        Bumping RBF Enabled Transaction - Transaction details of sent transaction. No bumping available. NOTE Transactions not sent with RBF enabled.
+    - image: /img/compatibility/copay/transaction-details-sent-warning.png
+      caption: >
+        Bumping RBF Enabled Transaction - Transactions details screen of sent transaction with “Amount too low to spend” warning message. Learn more link [goes here](https://support.bitpay.com/hc/en-us/articles/115004497783-What-does-the-BitPay-wallet-s-warning-amount-too-low-to-spend-mean-). Error message doesn’t make sense given a ~$7 transactions size. Note fee was 3 sat/byle.
```
Comment (adamjonas) [inline _data/compatibility/copay.yaml]: byle ->  byte

File: _data/compatibility/coinbase.yaml [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        Bumping RBF Transaction - Transaction not sent with RBF so no bumping possible.
+    - image: /img/compatibility/coinbase/transaction-details-sent-2.png
+      caption: >
+        Bumping RBF Transaction - After a period of time the View Transaction link shows up.
+    - image: /img/compatibility/coinbase/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction list. No RBF label.
+    - image: /img/compatibility/coinbase/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction details. No RBF label. Further transaction details are at blockcypher explorer which does not label RBF transactions.
```
Comment (adamjonas) [inline _data/compatibility/coinbase.yaml]: believe blockcypher should be upper camel case like "BlockCypher"

File: _data/compatibility/coinbase.yaml [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        Coinbase does not have an explicit option for receiving to bech32.
+        Coinbase uses p2sh wrapped segwit addresses for receiving.
+    - image: /img/compatibility/coinbase/segwit/send-screen.png
+      caption: >
+        Coinbase allows sending to either wrapped or native segwit addresses.
+        There is also visual validation of the address format.
+    - image: /img/compatibility/coinbase/segwit/transaction-details-sent.png
+      caption: >
+        Transaction details screen shows the bech32 address. However, the link
+        for that address uses a block explorer (blockcypher) which shows an
```
Comment (adamjonas) [inline _data/compatibility/coinbase.yaml]: [same](https://github.com/bitcoinops/bitcoinops.github.io/pull/180/files#r311756235) as above

**Comment by adamjonas** [comment]
Minor stuff.

File: _data/compatibility/edge.yaml (hunk 2 above)
Comment (bitschmidty) [reply _data/compatibility/edge.yaml]: @adamjonas actually Edge uses the blockchair.com explorer.

File: _data/compatibility/copay.yaml (hunk 4 above)
Comment (bitschmidty) [reply _data/compatibility/copay.yaml]: good eye!

**Comment by bitschmidty** [comment]
Pushed a commit to address @adamjonas' (thank you sir!) feedback

**Comment by bitschmidty** [comment]
pushed commits with:

- changes to the captions prefixes made per #183 (comment)
- changes to caption wording based on Bitcoin Core wallet example
- moved images into /rbf/ sub folder to be consistent with /segwit/ usability images (and more clear)
- changes to 'na' vs 'untested' for rbf sending tests when service doesn’t send via rbf

**Comment by bitschmidty** [comment]
rebased on master to get the latest compatibility styling and templates

**Comment by bitschmidty** [comment]
rebased and pushed commits for image optimizations

**Comment by bitschmidty** [comment]
squashed commits to one per evaluated service

**Comment by bitschmidty** [comment]
pushed a commit with changes that removed segwitv1 usability examples, since we are not including the data for now

**Comment by bitschmidty** [comment]
@adamjonas looking to merge this one today. If you have a second for final review 

https://deploy-preview-180--bitcoinops.netlify.com/en/compatibility/

File: _data/compatibility/coinbase.yaml [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+        Bumping RBF Transaction - After a period of time the View Transaction link shows up.
+    - image: /img/compatibility/coinbase/rbf/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction list. No RBF label.
+    - image: /img/compatibility/coinbase/rbf/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction details. No RBF label. Further transaction details are at BlockCypher explorer which does not label RBF transactions.
+    - image: /img/compatibility/coinbase/rbf/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - After bumped transaction confirmed, the bumped transaction then shows up and is credited. Original transactions stays as pending (even after 100 confirmations).
```
Comment (adamjonas) [inline _data/compatibility/coinbase.yaml]: Nit: stays -> stay to agree with plural txs.

Comment (bitschmidty) [reply _data/compatibility/coinbase.yaml]: good catch. corrected.

File: _data/compatibility/conio.yaml [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+        Receiving Bumped RBF Transaction - Receiving bumped transaction shows “Incoming” amount as the sum of the original and replacement transaction.
+    - image: /img/compatibility/conio/rbf/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Both original and replacement transaction show in transaction list screen.
+    - image: /img/compatibility/conio/rbf/transaction-list-replacement-confirmed.png
+      caption: >
+        Receiving Bumped RBF Transaction - After replacement transaction confirms, the original is removed from the list.
+    - image: /img/compatibility/conio/rbf/transaction-details-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Additional transaction details available once transaction was confirmed.
```
Comment (adamjonas) [inline _data/compatibility/conio.yaml]: with no segwit section this comes up blank on the main page which is inconsistent with the other companies.


Screen Shot 2019-08-19 at 3 34 06 PM

Comment (bitschmidty) [reply _data/compatibility/conio.yaml]: @adamjonas I have not been able to test their wallet for the last couple months due to geographical restrictions. So we do not have data for them yet. 

Ive been working on gaining access and I did just get an iOS TestFlight invite. I will try to re-test to try and get their segwit tests in there.

Comment (harding) [reply _data/compatibility/conio.yaml]: Although testing Coinio would be great, as a general solution we could probably just skip including tools without data in the listing, e.g.:

```diff
diff --git a/en/compatibility.md b/en/compatibility.md
index b4a239b..0467cf1 100644
--- a/en/compatibility.md
+++ b/en/compatibility.md
@@ -72,8 +72,6 @@ h1, h2, h3, h4, h5, h6 { text-align: center; }
       {% include functions/compat-cell.md state=tool.segwit.features.send.bech32 anchor="#segwit-send-bech32" %}
       {% include functions/compat-cell.md state=tool.segwit.features.send.bech32_p2wsh anchor="#segwit-send-bech32_p2wsh" %}
       {% include functions/compat-cell.md state=tool.segwit.features.send.change_bech32 anchor="#segwit-send-change_bech32" %}
-    {% else %}
-      <td colspan="7" class="default"></td>
     {% endif %}
   </tr>
 {% endfor %}
```

Comment (bitschmidty) [reply _data/compatibility/conio.yaml]: @harding I had the idea that a consistent set of wallets/services would be a good idea. But I agree with you and @adamjonas that the empty row is probably worse than omitting missing tests. I opened https://github.com/bitcoinops/bitcoinops.github.io/pull/209 for this.

Comment (bitschmidty) [reply _data/compatibility/conio.yaml]: Pushed a commit with Conio bech32 test results and screenshots and rebased.

**Comment by jnewbery** [comment]
I've reviewed the Conio bech32 support. Looks good!

ACK aa0744127cd119e26e20ac38871d3ba8a894a77c

**Pull request merged**